pkg/utils/utils.go

```diff
@@ -38,7 +38,7 @@ func GetEnvsByPrefix(prefix string, strip bool) map[string]string {
 			if len(pair[1]) > 0 {
 				k := pair[0]
 				if strip {
-					k = strings.Split(pair[0], prefix+"_")[1]
```

Comment:
Is this function actually used anywhere? This seems like something viper+cobra should be doing for us

Reply:
It doesn't seem so. Brett was the author, @xbcsmith were you planning for this to be used? Otherwise can just remove it.

Reply:
This seems like a leftover from xunit2receipt. i vote we chop it out